tests/lib/helpers.py

```diff
@@ -17,6 +18,16 @@ def does_directory_contain_files(file_list, directory_path):
     return True
 
 
+def does_directory_contain_n_files(directory_path, number):
```

Comment:
Could you ensure this only checks the number of files, and doesn't include any subdirectories?

Reply:
Check!